pkg/test/framework/components/echo/kube/deployment.go

```diff
@@ -196,6 +196,10 @@ spec:
 {{- end }}
           - --version
           - "{{ $subset.Version }}"
+{{- if $.IsMultiVersion }}
+          - --istio-version
```

Comment:
Would it hurt to just always set this?

Reply:
Shouldn't make a difference since `$version` gets defaulted to `""` in the templating when running w/o versions and we also default flag to `""` on echo server, can remove the check :+1:

Reply:
+1 to removing the check.